### PR TITLE
Add support for specifying `latest` for version

### PR DESCRIPTION
**Pull request by craigdbarber**
Integration tests successfully passed: https://github.com/GoogleCloudPlatform/github-actions/actions/runs/32798521

Addresses: https://github.com/GoogleCloudPlatform/github-actions/issues/31

### CODE REVIEW

File: setup-gcloud/src/version-util.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ */
+
+import * as httpm from 'typed-rest-client/HttpClient';
+
+/**
+ * @returns The latest stable version of the gcloud SDK.
+ */
+export async function getLatestGcloudSDKVersion(): Promise<string> {
+  const client: httpm.HttpClient = new httpm.HttpClient(
+    'github-actions-setup-gcloud',
```
Comment (averikitsch) [inline setup-gcloud/src/version-util.ts]: Can we pass this in from the other fie where it is defined?

File: setup-gcloud/src/version-util.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  );
+  return client
+    .get(
+      'https://api.github.com/repos/GoogleCloudPlatform/cloud-sdk-docker/tags',
+    )
+    .then(res => {
```
Comment (averikitsch) [inline setup-gcloud/src/version-util.ts]: Can we change this to async/await pattern to stay consistent?

File: setup-gcloud/src/version-util.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 Google LLC
```
[the remark this answers is not in the document]
Comment (craigdbarber) [reply setup-gcloud/src/version-util.ts]: You caught me copying and pasting :)
Done.

File: setup-gcloud/src/version-util.ts (hunk 1 above)
Comment (craigdbarber) [reply setup-gcloud/src/version-util.ts]: Done.

File: setup-gcloud/__tests__/version-util.test.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 Google LLC
```
[the remark this answers is not in the document]
Comment (craigdbarber) [reply setup-gcloud/__tests__/version-util.test.ts]: Done.

File: setup-gcloud/src/version-util.ts (hunk 2 above)
Comment (craigdbarber) [reply setup-gcloud/src/version-util.ts]: I would love to, sadly the library we're using here doesn't support it, so we have to wrap around it :)

Comment (averikitsch) [reply setup-gcloud/src/version-util.ts]: I'm seeing async/await in their "docs" aka tests (https://github.com/microsoft/typed-rest-client/blob/master/test/tests/httptests.ts#L52). This code is just getting harder and harder to read.

Comment (craigdbarber) [reply setup-gcloud/src/version-util.ts]: Oh snap! You are correct. For some reason when I first looked into using the library I had decided it didn't support it. Glad to see that's not the case. Good catch!

**Pull request merged**